crates/fuel-core/src/combined_database.rs

```diff
@@ -105,6 +105,19 @@ impl CombinedDatabase {
         })
     }
 
+    /// A test-only temporary rocksdb database with given rewind policy.
+    #[cfg(feature = "rocksdb")]
+    pub fn from_state_rewind_policy(
+        state_rewind_policy: StateRewindPolicy,
+    ) -> DatabaseResult<Self> {
+        Ok(Self {
+            on_chain: Database::rocksdb_temp(state_rewind_policy)?,
+            off_chain: Database::rocksdb_temp(state_rewind_policy)?,
+            relayer: Database::rocksdb_temp(state_rewind_policy)?,
```

Comment:
```suggestion
            relayer: Default::default(),
```

We don't need state rewind for the relayer database because all data can be restored from L1 and we can't alter the data itself=)

Reply:
aca530adcb